code/game/machinery/computer/card.dm

```diff
@@ -36,7 +36,11 @@ GLOBAL_VAR_INIT(time_last_changed_position, 0)
 		"Research Director",
 		"Chief Medical Officer",
 		"Brig Physician", 
-		"Deputy")
+		"Deputy",
+		"Brig Physician",
+		"Gimmick",
+		"Barber",
+		"Stage Magician")

```

Comment:
Should loop through `/datum/job/gimmick/` subtypes in Initialize() and add each of their titles to this.

Reply:
seems a bit pointless. This is the blacklist for jobs, after all

Reply:
did it anyways